keps/0004-add-testing-infrastructure.md

```diff
@@ -116,15 +116,14 @@ These clusters can be started either as a part of CI jobs or maintained long ter
 
 ### CICD
 
-Use [CircleCI](https://circleci.com/docs/) and the [GitHub Plugin](https://github.com/marketplace/circleci/plan/MDIyOk1hcmtldHBsYWNlTGlzdGluZ1BsYW45MA==#pricing-and-setup).
-
-For OpenSource projects we will receive 1,000 monthly build minutes. With the test suite below, that should suffice as a baseline.
+We will use a [Prow](https://github.com/kubernetes/test-infra/tree/master/prow) cluster to run jobs for pull requests and merges.
 
 #### Pull Requests
 
 All Pull Requests into master need to have the following checks pass. These should be ordered in fastest to slowest to reduce the time spent when/if failures occur.
 
 0. Check author has signed CLA
+1. If the user is not a contributor, a contributor must write `/ok-to-test` on the pull request before it will be triggered.
```

Comment:
^ -  Kudo community / contributors might want to pitch in on this. 

cc: @gerred

Reply:
I believe this is a security requirement, as otherwise anyone can run whatever code they want to in our Kubernetes cluster just by opening a pull request.

Reply:
This is usually the case for large projects and when tests are run against physical infrastructure, such as launching aws machines.

For most open source projects. I have seen the tests are the automatically triggered, and situation is optimized for immediate feedback without human intervention. 

I just thought that kudo contributors might have an some views/thoughts on this behavior.